file_scanning/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"ministryofjustice/opg-digideps/file_scanning/clamd"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+var opts map[string]string
+
+func init() {
+	log.SetOutput(ioutil.Discard)
+}
+
+// This is where the action happens.
+func scanHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	//POST takes the uploaded file(s) and saves it to disk.
```

Comment:
Should be a space after the `//` for the comment `// POST...`

`(can be ignored)`